packages/cubejs-testing/test/events-only-test-case.ts

```diff
@@ -0,0 +1,265 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Please rename to `driver-test-case.ts` and drastically simplify, see comment below.

Reply:
Done in 8cf68468d176df69bd5b7df7d8eae927f76bf858